internal/db/changelog.go

```diff
@@ -0,0 +1,31 @@
+package db
+
+import (
+	"fmt"
+	// Blank import required for mysql driver
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/jimmyjames85/bouncecm/internal/models"
+)
+
+// Changelog - Function to pull all rules from db
+func  (c *Client) Changelog() (models.ChangelogTable, error) {
+	rules := []models.ChangelogEntry{}
+
+	rows, err := c.Conn.Query("SELECT * FROM changelog ")
+
+
+	checkErr(err)

```

Comment:
would be better instead of check(Err) to include the code ```if err != nil {
		log.Println(err)
		return nil, err;
	}``` which will let us buble up aswell. the only reason we have checkErr() was to avoid writing that code over again.